### PR TITLE
Some Build Adjustments

### PR DESCRIPTION
**Pull request by orangewolf**
- fill in java_opts by default
- remove need for sed now that ruby is fixed in hyrax image
- update hyrax-doi branch to main as hyrax_upgrade is merged

@samvera/hyku-code-reviewers

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -10,7 +10,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2023_02_08_001228) do
+ActiveRecord::Schema.define(version: 2023_01_31_202855) do
```
Comment (alishaevn) [inline db/schema.rb:13]: why revert the schema?

File: .env [hunk 2]
```diff
@@ -13,6 +13,7 @@ FCREPO_PORT=8080
 FCREPO_REST_PATH=rest
 INITIAL_ADMIN_EMAIL=admin@example.com
 INITIAL_ADMIN_PASSWORD=testing123
+JAVA_OPTS=-Xmx4g -Xms1g
```
Comment (alishaevn) [inline .env:16]: should this include a space?

File: docker-compose.yml [hunk 3]
```diff
@@ -132,6 +129,8 @@ services:
 
   web:
     <<: *app
+    # Uncommend command to access container with out starting Rails. Useful for debugging
```
Comment (alishaevn) [inline docker-compose.yml:132]:
```suggestion
    # Uncomment command to access container with out starting Rails. Useful for debugging
```

**Review by alishaevn — CHANGES_REQUESTED** [review]

File: .env (hunk 2 above)
Comment (orangewolf) [reply .env:16]: yes, it needs the space

File: db/schema.rb (hunk 1 above)
Comment (orangewolf) [reply db/schema.rb:13]: i dont see a migration here. I dont know how the schema got rolled forward but the number seems wrong

**Pull request merged**